areas-of-d-usage.dd

```diff
@@ -36,8 +36,6 @@ $(DIVC start-icons,
 )
 
 $(DIV,
-    <img id="dman" src="images/d3.png" border="0" align="left" alt="D Man">
```

Comment:
I like it too, but that doesn't mean we should use it on every page. See commit message.